public/apps/configuration/panels/internal-user-edit/internal-user-edit.tsx

```diff
@@ -155,24 +168,34 @@ export function InternalUserEdit(props: InternalUserEditDeps) {
         setDataSource={setDataSource}
         selectedDataSource={dataSource}
       />
-      {props.buildBreadcrumbs(TITLE_TEXT_DICT[props.action])}
-      <EuiSpacer />
-      <EuiPageHeader>
-        <EuiFlexGroup direction="column" gutterSize="xs">
-          <EuiFlexItem>
-            <EuiTitle size="l">
-              <h1>{TITLE_TEXT_DICT[props.action]}</h1>
-            </EuiTitle>
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiText size="xs" color="subdued">
-              The security plugin includes an internal user database. Use this database in place of,
-              or in addition to, an external authentication system such as LDAP or Active Directory.{' '}
-              <ExternalLink href={DocLinks.UsersAndRolesDoc} />
-            </EuiText>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </EuiPageHeader>
+      <PageHeader
+        navigation={props.depsStart.navigation}
+        coreStart={props.coreStart}
+        descriptionControls={descriptionData}
+        fallBackComponent={
+          <>
+            <EuiSpacer />
+            <EuiPageHeader>
+              <EuiFlexGroup direction="column" gutterSize="xs">
+                <EuiFlexItem>
+                  <EuiTitle size="l">
+                    <h1>{TITLE_TEXT_DICT[props.action]}</h1>
+                  </EuiTitle>
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiText size="xs" color="subdued">
+                    The security plugin includes an internal user database. Use this database in
```

Comment:
Nit: Similar to previous comments on text. This is duplicated from above and can be extracted to a constants file

Reply:
This is a good idea, but i will table it for a follow up PR to limit the scope of this change. https://github.com/opensearch-project/security-dashboards-plugin/issues/2091